osmaxx/excerptexport/templates/excerptexport/email/all_exports_of_extraction_order_done_body.txt

```diff
@@ -1,10 +1,14 @@
-{% load navigation %}
-The extraction order #{{ extraction_order.id }} "{{ extraction_order.excerpt_name }}" has been processed.
-{% if successful_exports.count > 0 %}Results available for download:
-{% for export in successful_exports %}- {{ export.get_file_format_display }}{% if export.output_file.file %} ({{ export.output_file.file.url | siteabsoluteurl:request }}){% endif %}
+{% load navigation %}This is an automated email from the OSMaxx service.
```

Comment:
@sfkeller This line does not correspond to the [test](https://github.com/geometalab/osmaxx/pull/751/files#diff-426ecb9a4b4ee173d1345aa0f7813f29R259) below.